### PR TITLE
Projects redesign/project details

### PR DESCRIPTION
**Pull request by sunilsabatp**
This PR takes care the Project info section which includes
- project card
- about us
- project review
- project info
- contact details
- dive to map (only for mobile)

how to test ?
visit ->` /locale/prd/[project slug]`

- [x] project details sidebar (base PR) 
   - [x] conservation
   - [x] restoration - tree
   - [x] restoration - m2
- [x] unsupported project types (bouquet, funding) redirect to /prd with an error
- [x] site functionality 
   - [x] site selection on map
   - [x] dropdown
   - [x] zoom into site
- [ ] mobile controls
   - [ ] **explore (to check layout, comment out before release) - carried forward**
   - [x] sites dropdown 
- [x] plant location functionality (excluding selection on mobile)
- [x] sample plant location functionality (temporary)
- [x] show satellite layer by default if no plant locations exist
- [x] hide plant locations if satellite view is selected
- [ ] **static image of site on map (backend?) - carried forward**
- [x] ~~use existing slide up view (mobile) for plant location details --- cancelled~~
- [x] direct url visit with ploc parameter (in review - plant location PR #2211)
- [x] direct url visit with site parameter (in review - plant location PR #2211)
- [ ] **sample plant location/ plant location functionality (swipeable layout) - carried forward**
- [ ] **The back button logic does not seem right, perhaps this can be taken up in a separate PR though. - carried forward**
- [x] Bug on safari iOS - nav controls are overlapped by browser address bar (in review - this PR)
- [ ] **Evaluate styles/classes defined for KeyInfo / AdditionalInfo. For example, we have no class defined called `infoDetail` but this is used in AdditionalInfo, and KeyInfo has code which results in nested `singleRowInfoContainer` classes. Refactor, and rename where needed. (sep PR) - carried forward**
- [ ] **consolidate translations - carried forward**
- [x] consolidate utils
- [ ] **Consolidated UI check - list page - carried forward**
   - [ ] **mobile** 
   - [ ] **desktop**
- [ ] **Consolidated UI check - details page - carried forward**
   - [ ] **mobile**
   - [ ] **desktop** 
- [x] Handle projects without sites e.g. agroforestry-to-assist-forest-restoration-in-the-dry-bolivian-andes
   - [x] zoomToSingleTreeLocation is already defined
- [x] show whole area functionality for sample plant location info
- [ ] **Code cleanup - carried forward**
   - [ ] l**ogic to update url params for site/ploc, set selected plant locations, selected sites, selected plant locations etc. exists in many places, make this easier to understand and manage - carried forward**
   - [ ] **Check for references to old project code and separate where needed - carried forward**
   - [ ] **import types explicitly (consistent-type-imports) - carried forward**
- [ ] **I noticed that multiple calls are made to the plantLocations API in mobile devices (map mode) - perhaps this is happening whenever the url gets updated. - carried forward**
- [ ] **Handle case of setting site id to 0 when project has no sites. example (mothersforest project  unselecting a plant location; ploc param remains in the url) - carried forward**

### CODE REVIEW

**Comment by mariahosfeld** [comment]
Check again for design consistency.
On mobile: Some components are not taking the same width as others, e.g., the image slider and the video are more width.

**Review by mariahosfeld — COMMENTED** [review]
Looks good to merge. Once in complete context we will need to review fontsizes, some seem quite small.

**Comment by mohitb35** [comment]
UI
1. The trees donated count should be formatted into millions once it crosses a million. Also, only integer value should be seen while displaying the whole count.
e.g. 1 **26,551,073.21 Trees donated** ---> **26.5 million trees donated** or **26.5m Trees donated** 
e.g. 2 **3,392.2 Trees donated** --> **3,392 Trees donated**

**Comment by mohitb35** [comment]
Mobile specific UI (contd.)

2. What is to be shown as the background for the Dive into the project section.?

**Comment by mohitb35** [comment]
The price per unit shows up as 0 sometimes because the projects API is being called with a blank currency.

**Comment by mariahosfeld** [comment]
I checked for the unsupportet project types, we already have the same behavior as in our current live deployment.

**Comment by mariahosfeld** [comment]
Please compare the headlines for the project details of this PR with the live system. We made some adjustments for restoration projects, that need to reflect in this PR as well (e.g. First Tree Planted = Restoration Started). Use the wording that is on the live system.

**Comment by mariahosfeld** [comment]
For conservation projects some data is missing, please compare to the live system.

**Comment by mariahosfeld** [comment]
If no data for a box is available it should be hidden (e.g. the box containing Main Challenge)

**Comment by mohitb35** [comment]
@mariahosfeld I think we need to list down information that should be seen in the project details for conservation and tree projects. We have 2 main sections (KeyInfo and AdditionalInfo), and currently KeyInfo is shown only for tree projects. I'm not sure that is correct.

File: src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+          siteOwnershipText={metadata.siteOwnerName}
+          siteOwnershipType={isTreeProject ? metadata.siteOwnerType : null}
+          causeOfDegradationText={
+            isTreeProject ? metadata.degradationCause : null
+          }
+          whyThisSiteText={metadata.motivation}
+          longTermProtectionText={metadata.longTermPlan}
+          acquiredSince={metadata.acquisitionYear}
+        />
+      )}
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx:136]: 1. Need to clarify all information that can be seen in KeyInfo and AdditionalInfo sections, and update accordingly.
2. Conditions to render these sections will also need updating in that case.

File: src/features/projectsV2/ProjectDetails/components/KeyInfo.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/KeyInfo.tsx:1]: Need to evaluate the information shown in this component

File: src/features/projectsV2/ProjectDetails/components/microComponents/PlantingSeasons.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/microComponents/PlantingSeasons.tsx:1]: The `seasons` array of objects could be a simple array of translated month names, I don't see a reason to keep the current structure.

File: src/features/projectsV2/ProjectDetails/components/AdditionalInfo.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/AdditionalInfo.tsx:1]: We need to evaluate the information displayed in this section

File: src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import styles from '../styles/ProjectInfo.module.scss';
+import { useTranslations } from 'next-intl';
+import SingleProjectInfoItem from './microComponents/SingleProjectInfoItem';
+import ExternalCertificationItem from './microComponents/ExternalCertificationItem';
+import ProjectSpendingItem from './microComponents/ProjectSpendingItem';
+
+interface ExternalCertificationProps {
+  certification: Certificate[];
+  spendings: ProjectExpense[];
+}
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx:11]: 1. Let's match the prop names for ProjectDownloads with those returned in the API. Go down the component heirarchy while updating the names.
2. Prop interface should be just `Props` or `ProjectDownloadsProps` instead of the `ExternalCertificationProps` which is only a part of the functionality.

File: src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+const ProjectDownloads = ({
+  certification,
+  spendings,
+}: ExternalCertificationProps) => {
+  const tManageProjects = useTranslations('ManageProjects');
+
+  const certificationContent = [
+    {
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx:20]: Needs to be named more relevantly

File: src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx:1]: Review the names of props, variables, classes in this file. They do not seem to match with the intended purpose, or were created for another purpose which is no longer applicable.

File: src/features/projectsV2/ProjectDetails/components/microComponents/ProjectSpendingItem.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import DownloadsButton from './DownloadButton';
+import DownloadsLabel from './DownloadsLabel';
+import { getPDFFile } from '../../../../../utils/getImageURL';
+
+interface Props {
+  spendings: ProjectExpense[];
+}
+
+const ProjectSpendingItem = ({ spendings }: Props) => {
+  const screenWidth = window.innerWidth;
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/microComponents/ProjectSpendingItem.tsx:15]: Rename as `ProjectExpenseReports` or `ProjectExpenses`

File: src/features/projectsV2/ProjectDetails/components/microComponents/ExternalCertificationItem.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/microComponents/ExternalCertificationItem.tsx:1]: Update the filename to `ExternalCertificationItems`, and adjustment places where this component is imported as well.

File: src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import { Certificate, ProjectExpense } from '@planet-sdk/common';
+import styles from '../styles/ProjectInfo.module.scss';
+import { useTranslations } from 'next-intl';
+import SingleProjectInfoItem from './microComponents/SingleProjectInfoItem';
+import ExternalCertificationItem from './microComponents/ExternalCertificationItem';
+import ProjectSpendingItem from './microComponents/ProjectSpendingItem';
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx:6]: These are multiple items. Should be named accordingly.

File: src/features/projectsV2/ProjectDetails/components/microComponents/SingleContactDetail.tsx [hunk 6]
```diff
@@ -1,16 +1,20 @@
...
+  };
 }
 
 const SingleContactDetail = ({ contactInfo }: Props) => {
   return (
     <a
-      href={contactInfo.link}
-      target="_blank"
+      href={contactInfo.link || undefined}
+      target={contactInfo.title === 'View Profile' ? '_self' : '_blank'}
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/microComponents/SingleContactDetail.tsx:17]: I'm not in favour of keeping logic specifically for an expected value of contactInfo.title.

Perhaps add another prop `shouldOpenNewTab` - default could be `false`, and we can explicitly pass `true` in when the link needs to open in another tab.

**Review by mohitb35 — COMMENTED** [review]
@sunilsabatp Please go through my feedback.

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@sunilsabatp Please go through my feedback.

File: src/features/projectsV2/ProjectListControls/ProjectListControlForMobile.tsx [hunk 7]
```diff
@@ -86,15 +87,27 @@ const ProjectListControlForMobile = ({
...
+  const { activeSearchFieldClass, inActiveSearchFieldClass } = useMemo(() => {
+    return {
+      activeSearchFieldClass: isImpersonationModeOn
+        ? styles.tabsContainerTopMargin
+        : styles.searchFieldAndViewTabsContainer,
+      inActiveSearchFieldClass: isImpersonationModeOn
+        ? styles.projectListControlsMobileTopMargin
+        : styles.projectListControlsMobile,
+    };
+  }, [isImpersonationModeOn]);
```
[the remark this answers is not in the document]
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectListControls/ProjectListControlForMobile.tsx:100]: it's been simplified

File: src/features/projectsV2/ProjectsMap/index.tsx [hunk 8]
```diff
@@ -52,15 +51,12 @@ function ProjectsMap(props: ProjectsMapProps) {
...
   };
 
   return (
     <>
-      {props.isMobile && (
-        <div className={styles.projectListControlsContainer}>
-          <ProjectListControlForMobile {...projectListControlProps} />
-        </div>
-      )}
+      <MobileControls {...projectListControlProps} />
```
[the remark this answers is not in the document]
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectsMap/index.tsx:59]: I have refactored this component further in one of the sub-branches. Instead of using `<MobileControl>`, we will now use` <MapControls>`, which will serve as a common component for both mobile and desktop.

File: src/features/projectsV2/ProjectDetails/index.tsx [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+  const { setErrors, redirect } = useContext(ErrorHandlingContext);
+  const { tenantConfig } = useTenant();
+  const locale = useLocale();
+  const router = useRouter();
+
+  useEffect(() => {
+    if (setSelectedMode) setSelectedMode('list');
+    setSelectedClassification([]);
+    setDebouncedSearchValue('');
+  }, []);
```
[the remark this answers is not in the document]
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectDetails/index.tsx:40]: This logic was written for testing purposes, but I forgot to remove it

File: src/features/projectsV2/ProjectDetails/ProjectDetails.module.scss [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+.projectInfoSkeleton {
+  height: 100%;
+}
+
+.imageSliderContainer {
+  position: relative;
+  > button {
+    position: absolute;
+    z-index: 1000;
+    width: 18px;
```
[the remark this answers is not in the document]
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectDetails/ProjectDetails.module.scss:52]: The z-index value was necessary in order for the cross icon to appear on top of the image slider.

File: src/features/projectsV2/ProjectListControls/styles/ProjectListControls.module.scss [hunk 11]
```diff
@@ -20,17 +20,31 @@
...
 .searchFieldAndViewTabsContainer {
   display: flex;
   justify-content: space-between;
   gap: 4px;
+  padding: 0px 10px;
+}
+
+.tabsContainerTopMargin {
+  margin-top: 10px;
 }
```
[the remark this answers is not in the document]
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectListControls/styles/ProjectListControls.module.scss:48]: This styling has been simplified

**Comment by mariahosfeld** [comment]
Please make sure that all datapoint available via the API response are shown in the project details. For a conservation project several are missing.

**Comment by mariahosfeld** [comment]
For Site Ownership it needs to be ensured that we have an automatic break point if the space runs out, currently it is just cut off
image

File: src/features/projectsV2/ProjectDetails/index.tsx (hunk 9 above)
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/projectsV2/ProjectDetails/index.tsx]: This still happens - 

> When the user is in map mode on the list page, and comes to this page, they remain in map mode.

File: src/features/projectsV2/ProjectDetails/ProjectDetails.module.scss (hunk 10 above)
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/projectsV2/ProjectDetails/ProjectDetails.module.scss]: Yes, but using a figure like 1000 means that I need to use a random value above 1000 for any element that appears above this. Ideally there should be some logic to selecting a z-index value.

For example, selecting 0, 100, 200, 300 depending on the desired order of stacking elements, and using one standard high value (e.g. 9999) for elements that should always be at the top.

File: src/features/projectsV2/ProjectDetails/ProjectDetails.module.scss [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+  flex-direction: column;
+  grid-area: 15px;
+  gap: 15px;
+  box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.1);
+  @include xsPhoneView {
+    box-shadow: none;
+    margin-left: 0px;
+    padding: 15px 30px;
+  }
+}
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/ProjectDetails.module.scss:41]: Move these styles to ProjectInfo.module.scss - that makes more sense in terms of heirarchy, as ProjectInfoSection lies within the /components folder which uses styles from /styles.

File: src/features/projectsV2/ProjectDetails/index.tsx [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+        page="project-details"
+      />
+      <ProjectInfoSection
+        project={singleProject}
+        isMobile={isMobile}
+        setSelectedMode={setSelectedMode}
+      />
+    </div>
+  ) : (
+    <Skeleton className={styles.projectInfoSkeleton} />
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/index.tsx:91]: Since we have a ProjectInfoSection component, the `projectInfoSkeleton` class could be renamed to `projectDetailsSkeleton` as that is what it appears as a placeholder for

File: src/features/projectsV2/ProjectListControls/styles/ProjectListControls.module.scss [hunk 14]
```diff
@@ -20,21 +19,28 @@
...
+  gap: 10px;
+}
+
+.marginTop10 {
+  margin-top: 10px;
+}
+
+.marginTop20 {
+  margin-top: 20px;
 }
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectListControls/styles/ProjectListControls.module.scss:44]: 1. Still not sure why we need 2 different margins. 
2. Also, while using utility classes is fine, using a mixture of utility classes and other classes can cause confusion. In this scenario, I feel it would be better to use a specific class for impersonation mode.

File: src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+interface Props {
+  certificates: Certificate[];
+  expenses: ProjectExpense[];
+}
+
+const ProjectDownloads = ({ certificates, expenses }: Props) => {
+  const tManageProjects = useTranslations('ManageProjects');
+
+  const financialAndCertificationData = [
+    {
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx:17]: Let's rename `financelaAndCertificationData` --> `downloadsConfig`. 

It does not represent simple data, but a configuration of what should render and when, and we can also add another item (e.g. progressReports) without having to rename everything to make sense.

File: src/features/projectsV2/ProjectsContext.tsx [hunk 16]
```diff
@@ -58,6 +62,9 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
   setSelectedMode,
 }) => {
   const [projects, setProjects] = useState<MapProject[] | null>(null);
+  const [singleProject, setSingleProject] = useState<ProjectExtend | null>(
+    null
+  );
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectsContext.tsx:67]: Update the ProjectExtend type here as well.

File: src/features/projectsV2/ProjectSnippet/microComponents/ImageSection.tsx [hunk 17]
```diff
@@ -45,9 +47,25 @@ const ImageSection = (props: ImageSectionProps) => {
...
     );
   };
+  const handleBackButton = useCallback((e: MouseEvent) => {
+    e.stopPropagation();
+    if (window.history.length > 0) {
+      window.history.back();
+    } else {
+      window.location.href = '/';
+    }
+  }, []);
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectSnippet/microComponents/ImageSection.tsx:57]: The back button logic does not seem right, perhaps this can be taken up in a separate PR though.

File: src/features/projectsV2/ProjectsContext.tsx [hunk 18]
```diff
@@ -180,7 +190,7 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
     }
 
     loadProjects();
-  }, [currencyCode, locale]);
+  }, [currencyCode, locale, page]);
   useEffect(() => {
```
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/projectsV2/ProjectsContext.tsx]: This is not happening any more, but there is a possibility that the user landed on the project details page, and projects were not loaded, and then appeared here.

In this case, the projects do not ever get loaded.

File: src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx [hunk 19]
```diff
@@ -0,0 +1,166 @@
...
+        (isTreeProject && metadata.firstTreePlanted) ||
+        (isTreeProject && metadata.plantingDensity) ||
+        (isTreeProject && metadata.maxPlantingDensity) ||
+        (isTreeProject &&
+          metadata.plantingSeasons &&
+          metadata.plantingSeasons.length > 0) ||
+        (isConservationProject &&
+          metadata.activitySeasons &&
+          metadata.activitySeasons.length > 0)
+    );
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx:60]: Can this be more readable, with less duplication?

File: src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx [hunk 20]
```diff
@@ -0,0 +1,166 @@
...
+        (isConservationProject &&
+          metadata.landOwnershipType &&
+          metadata.landOwnershipType?.length > 0) ||
+        (isTreeProject && metadata.degradationCause) ||
+        longTermPlan ||
+        acquisitionYear ||
+        motivation ||
+        (mainInterventions && mainInterventions?.length > 0)
+    );
+  }, [metadata]);
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx:87]: Can this be more readable?

File: src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx [hunk 21]
```diff
@@ -0,0 +1,166 @@
...
+    reviews,
+    description,
+    purpose,
+    expenses,
+    certificates,
+    unitType,
+  } = project;
+  const isTreeProject = purpose === 'trees';
+  const isConservationProject = purpose === 'conservation';
+  const isRestorationProject = purpose === 'trees' && unitType === 'm2';
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx:45]: This condition does not depend on `unitType`

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@sunilsabatp Please go through all open comments, including the ones from earlier

**Comment by sunilsabatp** [comment]
> For Site Ownership it needs to be ensured that we have an automatic break point if the space runs out, currently it is just cut off image

The feature is scrollable, allowing you to scroll if the content of the site ownership type overflows.

File: src/features/projectsV2/ProjectSnippet/microComponents/ImageSection.tsx (hunk 17 above)
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectSnippet/microComponents/ImageSection.tsx:57]: will be handle in a separate PR

Comment (mohitb35) [reply src/features/projectsV2/ProjectSnippet/microComponents/ImageSection.tsx:57]: Noted. Have made a note in https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2149 as well.

File: src/features/projectsV2/ProjectsContext.tsx (hunk 18 above)
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/projectsV2/ProjectsContext.tsx]: Have made a note in #2149

File: src/features/projectsV2/ProjectDetails/components/KeyInfo.tsx [hunk 22]
```diff
@@ -0,0 +1,149 @@
...
+  };
+  const shouldRenderInterventionDate =
+    (interventionStarted &&
+      typeof interventionStarted === 'string' &&
+      interventionStarted?.length > 0) ||
+    interventionStarted;
+
+  return (
+    <div className={styles.projectInfoContainer}>
+      <div className={styles.singleRowInfoContainer}>
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/KeyInfo.tsx:58]: Minor rename - could this be `keyInfoContainer` instead of `projectInfoContainer` - both because it contains the `keyInfo` and also because `projectInfoContainer` and `projectInfoSection` are similar names and could cause confusion.

File: src/features/projectsV2/ProjectsContext.tsx [hunk 23]
```diff
@@ -58,6 +62,9 @@ export const ProjectsProvider: FC<ProjectsProviderProps> = ({
   setSelectedMode,
 }) => {
   const [projects, setProjects] = useState<MapProject[] | null>(null);
+  const [singleProject, setSingleProject] = useState<ExtendedProject | null>(
+    null
+  );
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectsContext.tsx:67]: This does not get set to `null` while going back from a project. As a result, while opening another project details page, I initially see the UI of the old project. 

Note: see if there are other context values that should be unset while visiting or going back from a project.

File: src/features/projectsV2/ProjectsMap/ProjectPopup/index.tsx [hunk 24]
```diff
@@ -28,6 +35,7 @@ const ProjectPopup = ({ project, handlePopupLeave, visitProject }: Props) => {
         <ProjectSnippet
           project={project.properties}
           showTooltipPopups={false}
+          selectedMode={selectedMode}
         />
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectsMap/ProjectPopup/index.tsx:39]: Why do we need to pass `selectedMode` down here?

File: src/features/projectsV2/ProjectsMap/MultipleProjectsView.tsx [hunk 25]
```diff
@@ -43,7 +37,7 @@ const MultipleProjectsView = () => {
   return (
     <ProjectMarkers
       categorizedProjects={categorizedProjects}
-      selectedClassification={selectedClassification}
+      isMobile={isMobile}
     />
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectsMap/MultipleProjectsView.tsx:41]: Why was this needed?

File: src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx [hunk 26]
```diff
@@ -0,0 +1,208 @@
...
+          metadata.ownershipType,
+        ]
+      : [];
+
+    return (
+      generalConditions.some(Boolean) ||
+      treeProjectConditions.some(Boolean) ||
+      conservationProjectConditions.some(Boolean)
+    );
+  }, [metadata]);
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx:114]: Add some comments about the logic for shouldRenderAdditionalInfo and shouldRenderKeyInfo as it could be confusing if I revisit this code later.

File: src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx [hunk 27]
```diff
@@ -0,0 +1,208 @@
...
+        <KeyInfo
+          abandonment={isTreeProject ? metadata.yearAbandoned : null}
+          interventionStarted={
+            isTreeProject
+              ? metadata.firstTreePlanted
+              : metadata.startingProtectionYear
+          }
+          interventionSeasons={
+            isTreeProject ? metadata.plantingSeasons : metadata.activitySeasons
+          }
```
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/components/ProjectInfoSection.tsx:155]: I think that combining 2 different data e.g. firstTreePlanted and startingProtectionYear makes the code more complicated. We need to add conditions while displaying this to check the project type, instead of simply checking that the data is not null. Consider passing in each separate data point separately.

File: src/features/projectsV2/ProjectsMap/MultipleProjectsView.tsx (hunk 25 above)
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectsMap/MultipleProjectsView.tsx:41]: this prop has been removed

File: src/features/projectsV2/ProjectsMap/ProjectPopup/index.tsx (hunk 24 above)
Comment (sunilsabatp) [reply src/features/projectsV2/ProjectsMap/ProjectPopup/index.tsx:39]: This prop has been removed

File: src/features/projectsV2/ProjectDetails/stories/KeyInfo.stories.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/stories/KeyInfo.stories.tsx:1]: This needs to be updated as well (since KeyInfo was updated)

File: src/features/projectsV2/ProjectDetails/stories/AdditionalInfo.stories.tsx
Comment (mohitb35) [inline src/features/projectsV2/ProjectDetails/stories/AdditionalInfo.stories.tsx:1]: This needs to be updated as well (since AdditionalInfo was updated)

**Review by mohitb35 — APPROVED** [review]
